client/app/lounge/activity.tsx

```diff
@@ -1,7 +1,207 @@
+import { useState } from "react";
```

Comment:
The requirements for this issue were only the backend API. Please remove this file 😄 

The frontend form for this API is being made here if you are curious 😃 https://github.com/ubcesports/lounge-hub/pull/28. 

This was helpful tho to look at how the API was called on the frontend so I'm going to take note of this commit hash and perhaps use some of the code when I get to it on my PR above 😄